NavdataStructStart(navdata_demo_t,navdata_demo)
NavdataStructMember(navdata_demo_t,navdata_demo,uint16_t,uint16,tag)
NavdataStructMember(navdata_demo_t,navdata_demo,uint16_t,uint16,size)
NavdataStructMember(navdata_demo_t,navdata_demo,uint32_t,uint32,ctrl_state)
NavdataStructMember(navdata_demo_t,navdata_demo,uint32_t,uint32,vbat_flying_percentage)
NavdataStructMember(navdata_demo_t,navdata_demo,float32_t,float32,theta)
NavdataStructMember(navdata_demo_t,navdata_demo,float32_t,float32,phi)
NavdataStructMember(navdata_demo_t,navdata_demo,float32_t,float32,psi)
NavdataStructMember(navdata_demo_t,navdata_demo,int32_t,int32,altitude)
NavdataStructMember(navdata_demo_t,navdata_demo,float32_t,float32,vx)
NavdataStructMember(navdata_demo_t,navdata_demo,float32_t,float32,vy)
NavdataStructMember(navdata_demo_t,navdata_demo,float32_t,float32,vz)
NavdataStructMember(navdata_demo_t,navdata_demo,uint32_t,uint32,num_frames)
NavdataStructMember(navdata_demo_t,navdata_demo,uint32_t,uint32,detection_camera_type)
NavdataStructEnd(navdata_demo_t,navdata_demo)

NavdataStructStart(navdata_time_t,navdata_time)
NavdataStructMember(navdata_time_t,navdata_time,uint16_t,uint16,tag)
NavdataStructMember(navdata_time_t,navdata_time,uint16_t,uint16,size)
NavdataStructMember(navdata_time_t,navdata_time,uint32_t,uint32,time)
NavdataStructEnd(navdata_time_t,navdata_time)

NavdataStructStart(navdata_raw_measures_t,navdata_raw_measures)
NavdataStructMember(navdata_raw_measures_t,navdata_raw_measures,uint16_t,uint16,tag)
NavdataStructMember(navdata_raw_measures_t,navdata_raw_measures,uint16_t,uint16,size)
NavdataStructArray(navdata_raw_measures_t,navdata_raw_measures,uint16_t,uint16,NB_ACCS,raw_accs)
NavdataStructArray(navdata_raw_measures_t,navdata_raw_measures,int16_t,int16,NB_GYROS,raw_gyros)
NavdataStructArray(navdata_raw_measures_t,navdata_raw_measures,int16_t,int16,2,raw_gyros_110)
NavdataStructMember(navdata_raw_measures_t,navdata_raw_measures,uint32_t,uint32,vbat_raw)
NavdataStructMember(navdata_raw_measures_t,navdata_raw_measures,uint16_t,uint16,us_debut_echo)
NavdataStructMember(navdata_raw_measures_t,navdata_raw_measures,uint16_t,uint16,us_fin_echo)
NavdataStructMember(navdata_raw_measures_t,navdata_raw_measures,uint16_t,uint16,us_association_echo)
NavdataStructMember(navdata_raw_measures_t,navdata_raw_measures,uint16_t,uint16,us_distance_echo)
NavdataStructMember(navdata_raw_measures_t,navdata_raw_measures,uint16_t,uint16,us_courbe_temps)
NavdataStructMember(navdata_raw_measures_t,navdata_raw_measures,uint16_t,uint16,us_courbe_valeur)
NavdataStructMember(navdata_raw_measures_t,navdata_raw_measures,uint16_t,uint16,us_courbe_ref)
NavdataStructMember(navdata_raw_measures_t,navdata_raw_measures,uint16_t,uint16,flag_echo_ini)
NavdataStructMember(navdata_raw_measures_t,navdata_raw_measures,uint16_t,uint16,frame_number)
NavdataStructMember(navdata_raw_measures_t,navdata_raw_measures,uint16_t,uint16,nb_echo)
NavdataStructMember(navdata_raw_measures_t,navdata_raw_measures,uint32_t,uint32,sum_echo)
NavdataStructMember(navdata_raw_measures_t,navdata_raw_measures,int32_t,int32,alt_temp_raw)
NavdataStructMember(navdata_raw_measures_t,navdata_raw_measures,int16_t,int16,gradient)
NavdataStructEnd(navdata_raw_measures_t,navdata_raw_measures)

NavdataStructStart(navdata_phys_measures_t,navdata_phys_measures)
NavdataStructMember(navdata_phys_measures_t,navdata_phys_measures,uint16_t,uint16,tag)
NavdataStructMember(navdata_phys_measures_t,navdata_phys_measures,uint16_t,uint16,size)
NavdataStructMember(navdata_phys_measures_t,navdata_phys_measures,float32_t,float32,accs_temp)
NavdataStructMember(navdata_phys_measures_t,navdata_phys_measures,uint16_t,uint16,gyro_temp)
NavdataStructArray(navdata_phys_measures_t,navdata_phys_measures,float32_t,float32,NB_ACCS,phys_accs)
NavdataStructArray(navdata_phys_measures_t,navdata_phys_measures,float32_t,float32,NB_GYROS,phys_gyros)
NavdataStructMember(navdata_phys_measures_t,navdata_phys_measures,uint32_t,uint32,alim3V3)
NavdataStructMember(navdata_phys_measures_t,navdata_phys_measures,uint32_t,uint32,vrefEpson)
NavdataStructMember(navdata_phys_measures_t,navdata_phys_measures,uint32_t,uint32,vrefIDG)
NavdataStructEnd(navdata_phys_measures_t,navdata_phys_measures)

NavdataStructStart(navdata_gyros_offsets_t,navdata_gyros_offsets)
NavdataStructMember(navdata_gyros_offsets_t,navdata_gyros_offsets,uint16_t,uint16,tag)
NavdataStructMember(navdata_gyros_offsets_t,navdata_gyros_offsets,uint16_t,uint16,size)
NavdataStructArray(navdata_gyros_offsets_t,navdata_gyros_offsets,float32_t,float32,NB_GYROS,offset_g)
NavdataStructEnd(navdata_gyros_offsets_t,navdata_gyros_offsets)

NavdataStructStart(navdata_euler_angles_t,navdata_euler_angles)
NavdataStructMember(navdata_euler_angles_t,navdata_euler_angles,uint16_t,uint16,tag)
NavdataStructMember(navdata_euler_angles_t,navdata_euler_angles,uint16_t,uint16,size)
NavdataStructMember(navdata_euler_angles_t,navdata_euler_angles,float32_t,float32,theta_a)
NavdataStructMember(navdata_euler_angles_t,navdata_euler_angles,float32_t,float32,phi_a)
NavdataStructEnd(navdata_euler_angles_t,navdata_euler_angles)

NavdataStructStart(navdata_references_t,navdata_references)
NavdataStructMember(navdata_references_t,navdata_references,uint16_t,uint16,tag)
NavdataStructMember(navdata_references_t,navdata_references,uint16_t,uint16,size)
NavdataStructMember(navdata_references_t,navdata_references,int32_t,int32,ref_theta)
NavdataStructMember(navdata_references_t,navdata_references,int32_t,int32,ref_phi)
NavdataStructMember(navdata_references_t,navdata_references,int32_t,int32,ref_theta_I)
NavdataStructMember(navdata_references_t,navdata_references,int32_t,int32,ref_phi_I)
NavdataStructMember(navdata_references_t,navdata_references,int32_t,int32,ref_pitch)
NavdataStructMember(navdata_references_t,navdata_references,int32_t,int32,ref_roll)
NavdataStructMember(navdata_references_t,navdata_references,int32_t,int32,ref_yaw)
NavdataStructMember(navdata_references_t,navdata_references,int32_t,int32,ref_psi)
NavdataStructMember(navdata_references_t,navdata_references,float32_t,float32,vx_ref)
NavdataStructMember(navdata_references_t,navdata_references,float32_t,float32,vy_ref)
NavdataStructMember(navdata_references_t,navdata_references,float32_t,float32,theta_mod)
NavdataStructMember(navdata_references_t,navdata_references,float32_t,float32,phi_mod)
NavdataStructMember(navdata_references_t,navdata_references,float32_t,float32,k_v_x)
NavdataStructMember(navdata_references_t,navdata_references,float32_t,float32,k_v_y)
NavdataStructMember(navdata_references_t,navdata_references,uint32_t,uint32,k_mode)
NavdataStructMember(navdata_references_t,navdata_references,float32_t,float32,ui_time)
NavdataStructMember(navdata_references_t,navdata_references,float32_t,float32,ui_theta)
NavdataStructMember(navdata_references_t,navdata_references,float32_t,float32,ui_phi)
NavdataStructMember(navdata_references_t,navdata_references,float32_t,float32,ui_psi)
NavdataStructMember(navdata_references_t,navdata_references,float32_t,float32,ui_psi_accuracy)
NavdataStructMember(navdata_references_t,navdata_references,int32_t,int32,ui_seq)
NavdataStructEnd(navdata_references_t,navdata_references)

NavdataStructStart(navdata_trims_t,navdata_trims)
NavdataStructMember(navdata_trims_t,navdata_trims,uint16_t,uint16,tag)
NavdataStructMember(navdata_trims_t,navdata_trims,uint16_t,uint16,size)
NavdataStructMember(navdata_trims_t,navdata_trims,float32_t,float32,angular_rates_trim_r)
NavdataStructMember(navdata_trims_t,navdata_trims,float32_t,float32,euler_angles_trim_theta)
NavdataStructMember(navdata_trims_t,navdata_trims,float32_t,float32,euler_angles_trim_phi)
NavdataStructEnd(navdata_trims_t,navdata_trims)

NavdataStructStart(navdata_rc_references_t,navdata_rc_references)
NavdataStructMember(navdata_rc_references_t,navdata_rc_references,uint16_t,uint16,tag)
NavdataStructMember(navdata_rc_references_t,navdata_rc_references,uint16_t,uint16,size)
NavdataStructMember(navdata_rc_references_t,navdata_rc_references,int32_t,int32,rc_ref_pitch)
NavdataStructMember(navdata_rc_references_t,navdata_rc_references,int32_t,int32,rc_ref_roll)
NavdataStructMember(navdata_rc_references_t,navdata_rc_references,int32_t,int32,rc_ref_yaw)
NavdataStructMember(navdata_rc_references_t,navdata_rc_references,int32_t,int32,rc_ref_gaz)
NavdataStructMember(navdata_rc_references_t,navdata_rc_references,int32_t,int32,rc_ref_ag)
NavdataStructEnd(navdata_rc_references_t,navdata_rc_references)

NavdataStructStart(navdata_pwm_t,navdata_pwm)
NavdataStructMember(navdata_pwm_t,navdata_pwm,uint16_t,uint16,tag)
NavdataStructMember(navdata_pwm_t,navdata_pwm,uint16_t,uint16,size)
NavdataStructMember(navdata_pwm_t,navdata_pwm,uint8_t,uint8,motor1)
NavdataStructMember(navdata_pwm_t,navdata_pwm,uint8_t,uint8,motor2)
NavdataStructMember(navdata_pwm_t,navdata_pwm,uint8_t,uint8,motor3)
NavdataStructMember(navdata_pwm_t,navdata_pwm,uint8_t,uint8,motor4)
NavdataStructMember(navdata_pwm_t,navdata_pwm,uint8_t,uint8,sat_motor1)
NavdataStructMember(navdata_pwm_t,navdata_pwm,uint8_t,uint8,sat_motor2)
NavdataStructMember(navdata_pwm_t,navdata_pwm,uint8_t,uint8,sat_motor3)
NavdataStructMember(navdata_pwm_t,navdata_pwm,uint8_t,uint8,sat_motor4)
NavdataStructMember(navdata_pwm_t,navdata_pwm,float32_t,float32,gaz_feed_forward)
NavdataStructMember(navdata_pwm_t,navdata_pwm,float32_t,float32,gaz_altitude)
NavdataStructMember(navdata_pwm_t,navdata_pwm,float32_t,float32,altitude_integral)
NavdataStructMember(navdata_pwm_t,navdata_pwm,float32_t,float32,vz_ref)
NavdataStructMember(navdata_pwm_t,navdata_pwm,int32_t,int32,u_pitch)
NavdataStructMember(navdata_pwm_t,navdata_pwm,int32_t,int32,u_roll)
NavdataStructMember(navdata_pwm_t,navdata_pwm,int32_t,int32,u_yaw)
NavdataStructMember(navdata_pwm_t,navdata_pwm,float32_t,float32,yaw_u_I)
NavdataStructMember(navdata_pwm_t,navdata_pwm,int32_t,int32,u_pitch_planif)
NavdataStructMember(navdata_pwm_t,navdata_pwm,int32_t,int32,u_roll_planif)
NavdataStructMember(navdata_pwm_t,navdata_pwm,int32_t,int32,u_yaw_planif)
NavdataStructMember(navdata_pwm_t,navdata_pwm,float32_t,float32,u_gaz_planif)
NavdataStructMember(navdata_pwm_t,navdata_pwm,uint16_t,uint16,current_motor1)
NavdataStructMember(navdata_pwm_t,navdata_pwm,uint16_t,uint16,current_motor2)
NavdataStructMember(navdata_pwm_t,navdata_pwm,uint16_t,uint16,current_motor3)
NavdataStructMember(navdata_pwm_t,navdata_pwm,uint16_t,uint16,current_motor4)
NavdataStructMember(navdata_pwm_t,navdata_pwm,float32_t,float32,altitude_prop)
NavdataStructMember(navdata_pwm_t,navdata_pwm,float32_t,float32,altitude_der)
NavdataStructEnd(navdata_pwm_t,navdata_pwm)

NavdataStructStart(navdata_altitude_t,navdata_altitude)
NavdataStructMember(navdata_altitude_t,navdata_altitude,uint16_t,uint16,tag)
NavdataStructMember(navdata_altitude_t,navdata_altitude,uint16_t,uint16,size)
NavdataStructMember(navdata_altitude_t,navdata_altitude,int32_t,int32,altitude_vision)
NavdataStructMember(navdata_altitude_t,navdata_altitude,float32_t,float32,altitude_vz)
NavdataStructMember(navdata_altitude_t,navdata_altitude,int32_t,int32,altitude_ref)
NavdataStructMember(navdata_altitude_t,navdata_altitude,int32_t,int32,altitude_raw)
NavdataStructMember(navdata_altitude_t,navdata_altitude,float32_t,float32,obs_accZ)
NavdataStructMember(navdata_altitude_t,navdata_altitude,float32_t,float32,obs_alt)
NavdataStructMember(navdata_altitude_t,navdata_altitude,vector31_t,ardrone_autonomy::vector31,obs_x)
NavdataStructMember(navdata_altitude_t,navdata_altitude,uint32_t,uint32,obs_state)
NavdataStructMember(navdata_altitude_t,navdata_altitude,vector21_t,ardrone_autonomy::vector21,est_vb)
NavdataStructMember(navdata_altitude_t,navdata_altitude,uint32_t,uint32,est_state)
NavdataStructEnd(navdata_altitude_t,navdata_altitude)

NavdataStructStart(navdata_vision_raw_t,navdata_vision_raw)
NavdataStructMember(navdata_vision_raw_t,navdata_vision_raw,uint16_t,uint16,tag)
NavdataStructMember(navdata_vision_raw_t,navdata_vision_raw,uint16_t,uint16,size)
NavdataStructMember(navdata_vision_raw_t,navdata_vision_raw,float32_t,float32,vision_tx_raw)
NavdataStructMember(navdata_vision_raw_t,navdata_vision_raw,float32_t,float32,vision_ty_raw)
NavdataStructMember(navdata_vision_raw_t,navdata_vision_raw,float32_t,float32,vision_tz_raw)
NavdataStructEnd(navdata_vision_raw_t,navdata_vision_raw)

NavdataStructStart(navdata_vision_of_t,navdata_vision_of)
NavdataStructMember(navdata_vision_of_t,navdata_vision_of,uint16_t,uint16,tag)
NavdataStructMember(navdata_vision_of_t,navdata_vision_of,uint16_t,uint16,size)
NavdataStructArray(navdata_vision_of_t,navdata_vision_of,float32_t,float32,5,of_dx)
NavdataStructArray(navdata_vision_of_t,navdata_vision_of,float32_t,float32,5,of_dy)
NavdataStructEnd(navdata_vision_of_t,navdata_vision_of)

NavdataStructStart(navdata_vision_t,navdata_vision)
NavdataStructMember(navdata_vision_t,navdata_vision,uint16_t,uint16,tag)
NavdataStructMember(navdata_vision_t,navdata_vision,uint16_t,uint16,size)
NavdataStructMember(navdata_vision_t,navdata_vision,uint32_t,uint32,vision_state)
NavdataStructMember(navdata_vision_t,navdata_vision,int32_t,int32,vision_misc)
NavdataStructMember(navdata_vision_t,navdata_vision,float32_t,float32,vision_phi_trim)
NavdataStructMember(navdata_vision_t,navdata_vision,float32_t,float32,vision_phi_ref_prop)
NavdataStructMember(navdata_vision_t,navdata_vision,float32_t,float32,vision_theta_trim)
NavdataStructMember(navdata_vision_t,navdata_vision,float32_t,float32,vision_theta_ref_prop)
NavdataStructMember(navdata_vision_t,navdata_vision,int32_t,int32,new_raw_picture)
NavdataStructMember(navdata_vision_t,navdata_vision,float32_t,float32,theta_capture)
NavdataStructMember(navdata_vision_t,navdata_vision,float32_t,float32,phi_capture)
NavdataStructMember(navdata_vision_t,navdata_vision,float32_t,float32,psi_capture)
NavdataStructMember(navdata_vision_t,navdata_vision,int32_t,int32,altitude_capture)
NavdataStructMember(navdata_vision_t,navdata_vision,uint32_t,uint32,time_capture)
NavdataStructMember(navdata_vision_t,navdata_vision,velocities_t,ardrone_autonomy::vector31,body_v)
NavdataStructMember(navdata_vision_t,navdata_vision,float32_t,float32,delta_phi)
NavdataStructMember(navdata_vision_t,navdata_vision,float32_t,float32,delta_theta)
NavdataStructMember(navdata_vision_t,navdata_vision,float32_t,float32,delta_psi)
NavdataStructMember(navdata_vision_t,navdata_vision,uint32_t,uint32,gold_defined)
NavdataStructMember(navdata_vision_t,navdata_vision,uint32_t,uint32,gold_reset)
NavdataStructMember(navdata_vision_t,navdata_vision,float32_t,float32,gold_x)
NavdataStructMember(navdata_vision_t,navdata_vision,float32_t,float32,gold_y)
NavdataStructEnd(navdata_vision_t,navdata_vision)

NavdataStructStart(navdata_vision_perf_t,navdata_vision_perf)
NavdataStructMember(navdata_vision_perf_t,navdata_vision_perf,uint16_t,uint16,tag)
NavdataStructMember(navdata_vision_perf_t,navdata_vision_perf,uint16_t,uint16,size)
NavdataStructMember(navdata_vision_perf_t,navdata_vision_perf,float32_t,float32,time_szo)
NavdataStructMember(navdata_vision_perf_t,navdata_vision_perf,float32_t,float32,time_corners)
NavdataStructMember(navdata_vision_perf_t,navdata_vision_perf,float32_t,float32,time_compute)
NavdataStructMember(navdata_vision_perf_t,navdata_vision_perf,float32_t,float32,time_tracking)
NavdataStructMember(navdata_vision_perf_t,navdata_vision_perf,float32_t,float32,time_trans)
NavdataStructMember(navdata_vision_perf_t,navdata_vision_perf,float32_t,float32,time_update)
NavdataStructArray(navdata_vision_perf_t,navdata_vision_perf,float32_t,float32,NAVDATA_MAX_CUSTOM_TIME_SAVE,time_custom)
NavdataStructEnd(navdata_vision_perf_t,navdata_vision_perf)

NavdataStructStart(navdata_trackers_send_t,navdata_trackers_send)
NavdataStructMember(navdata_trackers_send_t,navdata_trackers_send,uint16_t,uint16,tag)
NavdataStructMember(navdata_trackers_send_t,navdata_trackers_send,uint16_t,uint16,size)
NavdataStructEnd(navdata_trackers_send_t,navdata_trackers_send)

NavdataStructStart(navdata_vision_detect_t,navdata_vision_detect)
NavdataStructMember(navdata_vision_detect_t,navdata_vision_detect,uint16_t,uint16,tag)
NavdataStructMember(navdata_vision_detect_t,navdata_vision_detect,uint16_t,uint16,size)
NavdataStructMember(navdata_vision_detect_t,navdata_vision_detect,uint32_t,uint32,nb_detected)
NavdataStructArray(navdata_vision_detect_t,navdata_vision_detect,uint32_t,uint32,NB_NAVDATA_DETECTION_RESULTS,type)
NavdataStructArray(navdata_vision_detect_t,navdata_vision_detect,uint32_t,uint32,NB_NAVDATA_DETECTION_RESULTS,xc)
NavdataStructArray(navdata_vision_detect_t,navdata_vision_detect,uint32_t,uint32,NB_NAVDATA_DETECTION_RESULTS,yc)
NavdataStructArray(navdata_vision_detect_t,navdata_vision_detect,uint32_t,uint32,NB_NAVDATA_DETECTION_RESULTS,width)
NavdataStructArray(navdata_vision_detect_t,navdata_vision_detect,uint32_t,uint32,NB_NAVDATA_DETECTION_RESULTS,height)
NavdataStructArray(navdata_vision_detect_t,navdata_vision_detect,uint32_t,uint32,NB_NAVDATA_DETECTION_RESULTS,dist)
NavdataStructArray(navdata_vision_detect_t,navdata_vision_detect,float32_t,float32,NB_NAVDATA_DETECTION_RESULTS,orientation_angle)
NavdataStructArray(navdata_vision_detect_t,navdata_vision_detect,matrix33_t,ardrone_autonomy::matrix33,NB_NAVDATA_DETECTION_RESULTS,rotation)
NavdataStructArray(navdata_vision_detect_t,navdata_vision_detect,vector31_t,ardrone_autonomy::vector31,NB_NAVDATA_DETECTION_RESULTS,translation)
NavdataStructArray(navdata_vision_detect_t,navdata_vision_detect,uint32_t,uint32,NB_NAVDATA_DETECTION_RESULTS,camera_source)
NavdataStructEnd(navdata_vision_detect_t,navdata_vision_detect)

NavdataStructStart(navdata_watchdog_t,navdata_watchdog)
NavdataStructMember(navdata_watchdog_t,navdata_watchdog,uint16_t,uint16,tag)
NavdataStructMember(navdata_watchdog_t,navdata_watchdog,uint16_t,uint16,size)
NavdataStructMember(navdata_watchdog_t,navdata_watchdog,int32_t,int32,watchdog)
NavdataStructEnd(navdata_watchdog_t,navdata_watchdog)

NavdataStructStart(navdata_adc_data_frame_t,navdata_adc_data_frame)
NavdataStructMember(navdata_adc_data_frame_t,navdata_adc_data_frame,uint16_t,uint16,tag)
NavdataStructMember(navdata_adc_data_frame_t,navdata_adc_data_frame,uint16_t,uint16,size)
NavdataStructMember(navdata_adc_data_frame_t,navdata_adc_data_frame,uint32_t,uint32,version)
NavdataStructArray(navdata_adc_data_frame_t,navdata_adc_data_frame,uint8_t,uint8,32,data_frame)
NavdataStructEnd(navdata_adc_data_frame_t,navdata_adc_data_frame)

NavdataStructStart(navdata_video_stream_t,navdata_video_stream)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,uint16_t,uint16,tag)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,uint16_t,uint16,size)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,uint8_t,uint8,quant)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,uint32_t,uint32,frame_size)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,uint32_t,uint32,frame_number)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,uint32_t,uint32,atcmd_ref_seq)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,uint32_t,uint32,atcmd_mean_ref_gap)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,float32_t,float32,atcmd_var_ref_gap)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,uint32_t,uint32,atcmd_ref_quality)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,uint32_t,uint32,out_bitrate)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,uint32_t,uint32,desired_bitrate)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,int32_t,int32,data1)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,int32_t,int32,data2)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,int32_t,int32,data3)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,int32_t,int32,data4)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,int32_t,int32,data5)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,uint32_t,uint32,tcp_queue_level)
NavdataStructMember(navdata_video_stream_t,navdata_video_stream,uint32_t,uint32,fifo_queue_level)
NavdataStructEnd(navdata_video_stream_t,navdata_video_stream)

NavdataStructStart(navdata_games_t,navdata_games)
NavdataStructMember(navdata_games_t,navdata_games,uint16_t,uint16,tag)
NavdataStructMember(navdata_games_t,navdata_games,uint16_t,uint16,size)
NavdataStructMember(navdata_games_t,navdata_games,uint32_t,uint32,double_tap_counter)
NavdataStructMember(navdata_games_t,navdata_games,uint32_t,uint32,finish_line_counter)
NavdataStructEnd(navdata_games_t,navdata_games)

NavdataStructStart(navdata_pressure_raw_t,navdata_pressure_raw)
NavdataStructMember(navdata_pressure_raw_t,navdata_pressure_raw,uint16_t,uint16,tag)
NavdataStructMember(navdata_pressure_raw_t,navdata_pressure_raw,uint16_t,uint16,size)
NavdataStructMember(navdata_pressure_raw_t,navdata_pressure_raw,int32_t,int32,up)
NavdataStructMember(navdata_pressure_raw_t,navdata_pressure_raw,int16_t,int16,ut)
NavdataStructMember(navdata_pressure_raw_t,navdata_pressure_raw,int32_t,int32,Temperature_meas)
NavdataStructMember(navdata_pressure_raw_t,navdata_pressure_raw,int32_t,int32,Pression_meas)
NavdataStructEnd(navdata_pressure_raw_t,navdata_pressure_raw)

NavdataStructStart(navdata_magneto_t,navdata_magneto)
NavdataStructMember(navdata_magneto_t,navdata_magneto,uint16_t,uint16,tag)
NavdataStructMember(navdata_magneto_t,navdata_magneto,uint16_t,uint16,size)
NavdataStructMember(navdata_magneto_t,navdata_magneto,int16_t,int16,mx)
NavdataStructMember(navdata_magneto_t,navdata_magneto,int16_t,int16,my)
NavdataStructMember(navdata_magneto_t,navdata_magneto,int16_t,int16,mz)
NavdataStructMember(navdata_magneto_t,navdata_magneto,vector31_t,ardrone_autonomy::vector31,magneto_raw)
NavdataStructMember(navdata_magneto_t,navdata_magneto,vector31_t,ardrone_autonomy::vector31,magneto_rectified)
NavdataStructMember(navdata_magneto_t,navdata_magneto,vector31_t,ardrone_autonomy::vector31,magneto_offset)
NavdataStructMember(navdata_magneto_t,navdata_magneto,float32_t,float32,heading_unwrapped)
NavdataStructMember(navdata_magneto_t,navdata_magneto,float32_t,float32,heading_gyro_unwrapped)
NavdataStructMember(navdata_magneto_t,navdata_magneto,float32_t,float32,heading_fusion_unwrapped)
NavdataStructMember(navdata_magneto_t,navdata_magneto,char,uint8,magneto_calibration_ok)
NavdataStructMember(navdata_magneto_t,navdata_magneto,uint32_t,uint32,magneto_state)
NavdataStructMember(navdata_magneto_t,navdata_magneto,float32_t,float32,magneto_radius)
NavdataStructMember(navdata_magneto_t,navdata_magneto,float32_t,float32,error_mean)
NavdataStructMember(navdata_magneto_t,navdata_magneto,float32_t,float32,error_var)
NavdataStructEnd(navdata_magneto_t,navdata_magneto)

NavdataStructStart(navdata_wind_speed_t,navdata_wind_speed)
NavdataStructMember(navdata_wind_speed_t,navdata_wind_speed,uint16_t,uint16,tag)
NavdataStructMember(navdata_wind_speed_t,navdata_wind_speed,uint16_t,uint16,size)
NavdataStructMember(navdata_wind_speed_t,navdata_wind_speed,float32_t,float32,wind_speed)
NavdataStructMember(navdata_wind_speed_t,navdata_wind_speed,float32_t,float32,wind_angle)
NavdataStructMember(navdata_wind_speed_t,navdata_wind_speed,float32_t,float32,wind_compensation_theta)
NavdataStructMember(navdata_wind_speed_t,navdata_wind_speed,float32_t,float32,wind_compensation_phi)
NavdataStructMember(navdata_wind_speed_t,navdata_wind_speed,float32_t,float32,state_x1)
NavdataStructMember(navdata_wind_speed_t,navdata_wind_speed,float32_t,float32,state_x2)
NavdataStructMember(navdata_wind_speed_t,navdata_wind_speed,float32_t,float32,state_x3)
NavdataStructMember(navdata_wind_speed_t,navdata_wind_speed,float32_t,float32,state_x4)
NavdataStructMember(navdata_wind_speed_t,navdata_wind_speed,float32_t,float32,state_x5)
NavdataStructMember(navdata_wind_speed_t,navdata_wind_speed,float32_t,float32,state_x6)
NavdataStructMember(navdata_wind_speed_t,navdata_wind_speed,float32_t,float32,magneto_debug1)
NavdataStructMember(navdata_wind_speed_t,navdata_wind_speed,float32_t,float32,magneto_debug2)
NavdataStructMember(navdata_wind_speed_t,navdata_wind_speed,float32_t,float32,magneto_debug3)
NavdataStructEnd(navdata_wind_speed_t,navdata_wind_speed)

NavdataStructStart(navdata_kalman_pressure_t,navdata_kalman_pressure)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,uint16_t,uint16,tag)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,uint16_t,uint16,size)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,float32_t,float32,offset_pressure)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,float32_t,float32,est_z)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,float32_t,float32,est_zdot)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,float32_t,float32,est_bias_PWM)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,float32_t,float32,est_biais_pression)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,float32_t,float32,offset_US)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,float32_t,float32,prediction_US)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,float32_t,float32,cov_alt)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,float32_t,float32,cov_PWM)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,float32_t,float32,cov_vitesse)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,bool_t,int32,bool_effet_sol)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,float32_t,float32,somme_inno)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,bool_t,int32,flag_rejet_US)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,float32_t,float32,u_multisinus)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,float32_t,float32,gaz_altitude)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,bool_t,int32,Flag_multisinus)
NavdataStructMember(navdata_kalman_pressure_t,navdata_kalman_pressure,bool_t,int32,Flag_multisinus_debut)
NavdataStructEnd(navdata_kalman_pressure_t,navdata_kalman_pressure)

NavdataStructStart(navdata_hdvideo_stream_t,navdata_hdvideo_stream)
NavdataStructMember(navdata_hdvideo_stream_t,navdata_hdvideo_stream,uint16_t,uint16,tag)
NavdataStructMember(navdata_hdvideo_stream_t,navdata_hdvideo_stream,uint16_t,uint16,size)
NavdataStructMember(navdata_hdvideo_stream_t,navdata_hdvideo_stream,uint32_t,uint32,hdvideo_state)
NavdataStructMember(navdata_hdvideo_stream_t,navdata_hdvideo_stream,uint32_t,uint32,storage_fifo_nb_packets)
NavdataStructMember(navdata_hdvideo_stream_t,navdata_hdvideo_stream,uint32_t,uint32,storage_fifo_size)
NavdataStructMember(navdata_hdvideo_stream_t,navdata_hdvideo_stream,uint32_t,uint32,usbkey_size)
NavdataStructMember(navdata_hdvideo_stream_t,navdata_hdvideo_stream,uint32_t,uint32,usbkey_freespace)
NavdataStructMember(navdata_hdvideo_stream_t,navdata_hdvideo_stream,uint32_t,uint32,frame_number)
NavdataStructMember(navdata_hdvideo_stream_t,navdata_hdvideo_stream,uint32_t,uint32,usbkey_remaining_time)
NavdataStructEnd(navdata_hdvideo_stream_t,navdata_hdvideo_stream)

NavdataStructStart(navdata_wifi_t,navdata_wifi)
NavdataStructMember(navdata_wifi_t,navdata_wifi,uint16_t,uint16,tag)
NavdataStructMember(navdata_wifi_t,navdata_wifi,uint16_t,uint16,size)
NavdataStructMember(navdata_wifi_t,navdata_wifi,uint32_t,uint32,link_quality)
NavdataStructEnd(navdata_wifi_t,navdata_wifi)

NavdataStructStart(navdata_zimmu_3000_t,navdata_zimmu_3000)
NavdataStructMember(navdata_zimmu_3000_t,navdata_zimmu_3000,uint16_t,uint16,tag)
NavdataStructMember(navdata_zimmu_3000_t,navdata_zimmu_3000,uint16_t,uint16,size)
NavdataStructMember(navdata_zimmu_3000_t,navdata_zimmu_3000,int32_t,int32,vzimmuLSB)
NavdataStructMember(navdata_zimmu_3000_t,navdata_zimmu_3000,float32_t,float32,vzfind)
NavdataStructEnd(navdata_zimmu_3000_t,navdata_zimmu_3000)

